### PR TITLE
Updated livesite with new background, logo, buttons, and new colors

### PR DESCRIPTION
**Pull request by jaytran505**
This is an initial update to the live site. It now includes the new background, logo, and some new buttons. Kalp and I also changed the background color and the text box colors. We also switched out the wild west font and got rid of the dusty and cactus stuff. These are just the first colors we randomly chose so we can change them if we want. some screenshots below:

image
image
image

### CODE REVIEW

**Comment by susanto-tm** [comment]
The workshops and FAQ can probably have some similar styles to the dialog box for the hero section just with a less fancy font. 

The rich text shouldn't have indents at the beginning since it's just a p tag with no inline styles.

File: user-registration-app/src/app/views/live-view/live-view.component.css [hunk 1]
```diff
@@ -17,11 +17,11 @@ many of the utlities in Bootstrap, so undoing it here.*/
 }
 .header {
   background-size: cover;
-  /*background-image: url('../../../assets/images/stars_1.svg') , linear-gradient(0deg ,#11334A 40%,#000000 100%);*/
-  background-repeat: no-repeat;
+  background-image: url('../../../assets/images/vapor-backgound.png');
```
Comment (susanto-tm) [inline user-registration-app/src/app/views/live-view/live-view.component.css:20]: Add `background-position: center`

File: user-registration-app/src/app/views/live-view/live-view.component.html [hunk 2]
```diff
@@ -12,18 +12,19 @@
...
             class="animated rubberBand d-block mx-auto mb-0"
-            alt="hackpsulogo"
+            alt="HackPSU logo with glitch effect"
+            width="60%"
           />
         </div>
 
         <!-- Countdown Timer -->
-        <div class="col-6">
+        <div class="col-6" style="padding-left: 10%">
```
Comment (susanto-tm) [inline user-registration-app/src/app/views/live-view/live-view.component.html:27]: remove `padding-left`

**Review by susanto-tm — CHANGES_REQUESTED** [review]
Remove this line to remove the rich text padding: https://github.com/Hack-PSU/frontend/blob/fbd02c56c591aec2e3ca55fdbc316faad8409a0d/user-registration-app/src/app/views/live-workshop/live-workshop.component.html#L16

**Comment by susanto-tm** [comment]
Also make the anchor links of the navbar have smooth scrolling

**Comment by jdurrell** [comment]
Poggers, guys. Merging this

**Pull request merged**